src/inversion_scripts/invert.py

```diff
@@ -253,6 +272,7 @@ def calculate_superobservation_error(sO, p):
     gamma = float(sys.argv[10])
     res = sys.argv[11]
     jacobian_sf = sys.argv[12]
+    prior_err_BC = float(sys.argv[13]) if len(sys.argv) > 13 else None
```

Comment:
I might be forgetting how `sys.argv` works, but just want to confirm the `>` condition is correct -- or should it be `>=` or `==` ?

Reply:
I believe that `>` is correct because `invert.py` is called either like:
`python invert.py $nElements $JacobianDir $posteriorSF $LonMinInvDomain $LonMaxInvDomain $LatMinInvDomain $LatMaxInvDomain $PriorError $ObsError $Gamma $Res $jacobian_sf`

(in which `len(sys.argv)` would be exactly 13)
or:

`python invert.py $nElements $JacobianDir $posteriorSF $LonMinInvDomain $LonMaxInvDomain $LatMinInvDomain $LatMaxInvDomain $PriorError $ObsError $Gamma $Res $jacobian_sf $PriorErrorBCs`
(in which `len(sys.argv)` would be 14)

The thing to remember is that `sys.argv[0]` is always the name of the python file (here `invert.py`).